lib/config_verify/util.go

```diff
@@ -0,0 +1,306 @@
+package config_verify
+
+import (
+	"container/list"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/chef/automate/lib/config_parser"
+	"github.com/chef/automate/lib/stringutils"
+)
+
+const (
+	FOR_CERTS = " for certs "
+)
+
+func validateAutomateFQDN(fqdn string) error {
```

Comment:
we already have one function in helper.go named validateFQDN. Can't we reuse that instead of writing the same code again.

Reply:
Yes, since since args and return types were different had created different function, but figured out a way make use of same code.